src/components/managersinglecomm/ManagerSingleCommunity.js

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image'
+import styles from './managersinglecomm.module.css'
+import CustomizableButton from '../common/CustomizableButton'
+
+function ManagerCommunity({ customClass }) {
+  return (
+    <main className={styles.card}>
+      <div className={styles['img-container']}>
```

Comment:
Hey @TaylorDJones11 so we decided that using `camelCase` in the CSS files is actually recommended for React. 😅 So please return to using `camelCase` for your CSS properties, as this will be our convention moving forward. 🙏🏾

Reply:
Everything else looks great; I will approve after CSS case changes